admin/app/assets/javascripts/workarea/admin/modules/selects.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @namespace WORKAREA.selects
+ */
+WORKAREA.registerModule('selects', (function () {
```

Comment:
(minor) I started with something similar but I thought it felt a little generic. 🤷‍♂